jans-cli-tui/setup.py

```
@@ -6,10 +6,24 @@
 import codecs
 import os
 import re
+import time
 from setuptools import setup
 from setuptools import find_packages
 from setuptools.command.install import install
 from urllib.request import urlretrieve
+from urllib.error import URLError
+
+def _download_with_retry(url, dest, retries=5, delay=10):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_download_with_retry`

Add return type annotation: `None`

(ANN202)